src/main/java/se/nackademin/Inventory.java

```diff
@@ -0,0 +1,80 @@
+package se.nackademin;
+
+import java.util.HashMap;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Inventory {
```

Comment:
Some confusion in this class. The word "Inventory" implies to me a table of many products, but the instance variables shows that this is more like one "row" in the table, e.g. the count for just one. Further down there is a `productMap` variable that keeps track of many rows again. Probably this class should continue to be a "row" of Inventory, e.g. an `InventoryRecord` or `StockRecord` and the `productMap` should be in Main.